containers/base/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
 
 # Setup Node.js
```

Comment:
I'm new to the Datalab development environment so my personal preference is to have more comments in the Datalab Dockerfile (and potentially external links in the comments to obtain more information). The following notes could be helpful for maintenance purposes:
- `Use the Node LTS build for stability reasons. See `https://nodejs.org/en/download/
- `Maintenance end date for Node 6.x LTS is 2019-04-18. See ` https://github.com/nodejs/LTS/